src/Service/ApiClient.php

```diff
@@ -0,0 +1,50 @@
+<?php
+declare (strict_types=1);
+
+namespace Service;
+
+
+class ApiClient {
+
+    /** @var \Maleficarum\Client\Http\Rest\BasicClient  */
+    protected $httpClient;
+
+    /**
+     * ApiClient constructor.
+     *
+     * @param \Maleficarum\Client\Http\Rest\BasicClient $httpClient
+     */
+    public function __construct(\Maleficarum\Client\Http\Rest\BasicClient $httpClient) {
+        $this->httpClient = $httpClient;
+    }
+
+    /**
+     * @param int $articleId
+     *
+     * @return array
+     */
+    public function getArticle(int $articleId): array {
+        $this->httpClient->get('/articles/'.$articleId);
+
+        return $this->httpClient->getParsedBody();
+    }
+
+    /**
+     * @param int $limit
+     * @param int $offset
+     *
+     * @return array
+     */
+    public function getArticles($limit = 10, $offset = 0): array {
+        $parameters = [
+            'limit' => $limit,
+            'offset' => $offset,
+            'sort' => '-id',
+            'status' => 1 //Only active
```

Comment:
parametry typu sort i status są świetnym przykładem na to że powinno być SDK. Załóżmy że API nie jest wewnętrznego autorstwa tylko jest to usługa zewnętrzna. Skąd ten biedny programista ma wiedzieć co to jest sort -`id` albo status 69?

dodatkowo jakbyś zrobił mape statusów to nie trzeba by było pisać komentarzy. Przykład
const STATUS_ACTIVE = 1;

to samo dla sorta:
const SORT_ID_DESC = '-id';
const SORT_ID_ASC = '+id';
...